app/test/unit/git/cherry-pick-test.ts

```diff
@@ -264,4 +271,126 @@ describe('git/cherry-pick', () => {
     }
     expect(result).toBe(null)
   })
+
+  describe('cherry picking with conflicts', () => {
+    beforeEach(async () => {
+      // In the 'git/cherry-pick' `beforeEach`, we call `createRepository` which
+      // adds a commit to the feature branch with a file called THING.md. In
+      // order to make a conflict, we will add the same file to the target
+      // branch.
+      const conflictingCommit = {
+        commitMessage: 'Conflicting Commit!',
+        entries: [
+          {
+            path: 'THING.md',
+            contents: '# HELLO WORLD! \n CREATING CONFLICT! FUN TIMES!\n',
```

Comment:
> CREATING CONFLICT! FUN TIMES!

![Some men just want to watch the world burn](https://media2.giphy.com/media/hcYUjaDFSa6WI/giphy-downsized.gif?cid=549b592d5d27c5a7414e637967ee5582&rid=giphy-downsized.gif)